code/modules/atmospherics/machinery/components/fusion/hfr_fuel_datums.dm

```diff
@@ -80,7 +80,7 @@ GLOBAL_LIST_INIT(hfr_fuels_list, hfr_fuels_create_list())
 	negative_temperature_multiplier = 2.5
 	positive_temperature_multiplier = 0.1
 	energy_concentration_multiplier = 10
-	fuel_consumption_multiplier = 0.3
+	fuel_consumption_multiplier = 2.5
```

Comment:
Where do these numbers actually come from? What about them makes this a bug fix and not a balance change?

Reply:
Since I inverted the equation, I had to invert the numbers too, but I now see it should have been a 3.3, dunno where 2.5 came from. I'll change it

Reply:
Is there any way you can make at least one of the numbers automatically derived?

Suppose I was balancing this feature--if I just changed this number, would some equation be broken?

Reply:
Ah nono, no equations would break, it will work as intended, increase the number and the fuel consumption will increase, decrease it and it will decrease

Reply:
Tho I might expand the docs to explain better what to do and what it does